packages/salesforcedx-vscode-soql/src/client/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * This file is intentionally left blank. Exports are exported
+ * in the root of salesforcedx-vscode-soql.
+ */
```

Comment:
Comments-only `index.ts` files like these in the subfolders prevent additional large webpacked modules from being produced.